src/main/scala/chisel3/experimental/inlinetest/package.scala

```diff
@@ -0,0 +1,80 @@
+package chisel3.experimental.inlinetest
```

Comment:
Nit, the name of this file shouldn't be `package.scala`, that is reserved for package objects, e.g. [[1]](https://github.com/chipsalliance/chisel/blob/1ac76764f2cfd2d37f5f25e4f9b42da665ffd5f0/core/src/main/scala-2/chisel3/probe/package.scala#L10).

I'm not sure what to call this file since it has basically everything in the package in it, maybe just `InlineTest.scala` (still in directory `src/main/scala/chisel3/experimental/inlinetest/`) and we can break it up later if we see fit?